tests/Module/Fixture/DisplayUserEvents.php

```diff
@@ -37,37 +37,54 @@ public function haveUserEvents()
             'leave',
             'sign up'
         ];
-
-        return $events;
     }
 
-    public function haveUserEventsForExperimentStats()
+    public function haveSummarizationEvents(int $ownerId, array $events)
```

Comment:
what? Is it summarizable events? 
Summarizing is a type of event, but here you depend on the type in $event['type'].

So, you don't have to specify it in method name.